### PR TITLE
fix #75006 cars.mclaren.com

### LINKED ISSUE #75006: cars.mclaren.com
### Issue URL (Annoyance)
 [https://cars.mclaren.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fcars.mclaren.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.0.3
Browser: | Safari
Simplified filters: | disabled
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances

### PR DESCRIPTION
**Pull request by wout-codes**
#75006
No breakage.

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_general.txt [hunk 1]
```diff
@@ -63,6 +63,7 @@
 ##.gpm-cookie-accepted
 ##cloudflare-app[app="cookiless"]
 ###privacy-consent
+##.visible[class*="cookie-alert"][data-component]
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_general.txt:66]: I think we should add domain-specific rule only

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 2]
```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+mclaren.com##.visible[class*="cookie-alert"][data-component]
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt:94]: imho too complex
mclaren.com##div[class*="-cookie-alert"] is enough

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Ok

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
@wout-codes also don't add your rules at the beginning. It causes conflicts, so add them a little lower.

**Pull request merged**